.github/workflows/auto-comment.yml

```
@@ -25,6 +25,8 @@ jobs:
 
             - [ ] Code is PEP8-compliant.
 
+            - [ ] Changelog or what's new entry
```

```suggestion
            - [ ] Changelog or what's new entry in "doc/changes/latest.rst"
```
This might help newcomers find the right file more quickly